src/numbuf/python/src/pynumbuf/numbuf.cc

```diff
@@ -275,12 +345,9 @@ static PyObject* store_list(PyObject* self, PyObject* args) {
   }
   CHECK(error_code == PlasmaError_OK);
 
-  auto target = std::make_shared<FixedBufferStream>(LENGTH_PREFIX_SIZE + data, size);
-  std::shared_ptr<arrow::ipc::FileWriter> writer;
-  ipc::FileWriter::Open(target.get(), batch->schema(), &writer);
-  writer->WriteRecordBatch(*batch, true);
-  writer->Close();
-  *((int64_t*)data) = size;
+  auto target = std::make_shared<FixedBufferStream>(LENGTH_PREFIX_SIZE + data, total_size);
```

Comment:
@wesm Thanks! I think we will need to modify the class so we can use our custom multithreaded memcpy. We could provide an interface similar to https://github.com/ray-project/ray/blob/master/src/numbuf/python/src/pynumbuf/memory.h#L19 and have a default one that just calls std::memcpy. What do you think about that?